src/bcrhp/bcrhp/migrations/665_4_public_internal_map_layers.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.2.13 on 2024-06-24 11:16
+
+from collections import namedtuple
+
+from django.core.management import call_command
+from django.db import migrations
+from django.db.migrations.operations.special import RunPython
+
+from bcrhp.settings import APP_ROOT
+
+mapbox_overlays_dir = f'{APP_ROOT}/pkg/map_layers/mapbox_spec_json/overlays'
+
+MapEntry = namedtuple('MapEntry', 'name file')
+map_layers = [
+    MapEntry('Borden Grid', 'borden-grid'),
+    MapEntry('Cadastral ParcelMap', 'cadastral'),
+    MapEntry('Crown Tenures', 'crown-tenures'),
+]
+
+def forward(apps, schema_editor):
+    for ml in map_layers:
+        call_command('packages', operation='delete_mapbox_layer', layer_name=ml.name)
+        call_command('packages', operation='add_mapbox_layer',
+                     layer_name=ml.name,
+                     mapbox_json_path=f'{mapbox_overlays_dir}/{ml.file}/{ml.file}.json')
+
+
+def backward(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+    MapLayer = apps.get_model('arches', 'MapLayer')
```

Comment:
Is there a reason you're not using the privileged SQL method for the backward migration? This also feels like the reverse should be the same as the forward but from the common package directory.

Reply:
The reason for not using the privileged sql is because of the way the migration operation objects work, since im running python code, the privileged sql class isn't suitable.

The reason for the django model usage is due to the nature of the json files being static. For reverting the data of the map layer, the json files would need to be the previous data but that doesn't work when you need to be on this commit for example to have the new changes to go forward.

Since the get_model call isn't working, what i might be able to do is call a raw execute sql function with the username and pass as what the privileged sql is doing.

Reply:
OK, great. Thank you for clarifying.